test/functional/sops_controller_test.rb

```diff
@@ -540,28 +540,102 @@ def test_editing_doesnt_change_contributor
   test 'should set the policy to projects_policy if the item is requested to be published, when creating new sop' do
```

Comment:
is the title of the test still correct? Wasn't it changed from setting it to the project policy?

Reply:
No, that change has not been implemented in this PR yet, so the title is still correct.

Reply:
It has now been added to the pull request... see [da17ad2](https://github.com/seek4science/seek/pull/1484/commits/da17ad26b756272d7ce18225ed0d113d5292458b).